Index: openwrt-builder/custom-feed/netbird/Makefile
===================================================================
--- openwrt-builder.orig/custom-feed/netbird/Makefile
+++ openwrt-builder/custom-feed/netbird/Makefile
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netbird
-PKG_VERSION:=0.29.0
+PKG_VERSION:=0.29.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/netbirdio/netbird/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=b1a76b2fcee9e8d1c38f59818c8226a7a0d991d7dca08a3a217bc9ce05ea8fef
+PKG_HASH:=9072d16845de49ce395b150564b82f97a33defe007276b6b46d733306b6760ee
 
 PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
 PKG_LICENSE:=BSD-3-Clause
