From 9fa1472aaf57d99720c1e232a40c3985364165fb Mon Sep 17 00:00:00 2001
From: Wesley Gimenes <wehagy@proton.me>
Date: Sun, 6 Oct 2024 07:06:43 -0300
Subject: [PATCH] netbird: update to 0.30.0

Signed-off-by: Wesley Gimenes <wehagy@proton.me>
---
 net/netbird/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/netbird/Makefile b/net/netbird/Makefile
index cc5fc0561..0af523ff0 100644
--- a/net/netbird/Makefile
+++ b/net/netbird/Makefile
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=netbird
-PKG_VERSION:=0.29.4
+PKG_VERSION:=0.30.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/netbirdio/netbird/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=a3ad5dd3deda26e7888706af99c2e093cc03c6db2d47cf3569c28b8306d23c2f
+PKG_HASH:=cb75e71f8f3b2f21272f38468d50464dd8d9de74b776e9c2f52ac440229453b2
 
 PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
 PKG_LICENSE:=BSD-3-Clause
